irohad/ametsuchi/impl/temporary_wsv_impl.cpp

```diff
@@ -40,14 +40,17 @@ namespace iroha {
       auto execute_command = [this, transaction](auto command) {
         auto executor = command_executors_->getCommandExecutor(command);
         auto account = wsv_->getAccount(transaction.creator_account_id).value();
-        return executor->validate(*command, *wsv_, account) &&
-            executor->execute(*command, *wsv_, *executor_);
+        return executor->validate(
+                   *command, *wsv_, transaction.creator_account_id)
+            && executor->execute(
+                   *command, *wsv_, *executor_, transaction.creator_account_id);
       };
 
       transaction_->exec("SAVEPOINT savepoint_;");
-      auto result = function(transaction, *wsv_) &&
-          std::all_of(transaction.commands.begin(),
-                      transaction.commands.end(), execute_command);
+      auto result = function(transaction, *wsv_)
```

Comment:
`function` name is quite unclear for me, so maybe give it a better name? What should it do?